### PR TITLE
Update location of Windows Service Console downloads

### PR DESCRIPTION
**Pull request by yatindrakulkarni-mendix**
Edit step to download mendix service console
Awaiting final confirmation before publishing. See https://github.com/mendix/docs/pull/7440#issuecomment-2019467261, below.

### CODE REVIEW

File: content/en/docs/developerportal/deploy/on-premises-design/ms-windows/_index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Before starting this how-to, make sure you have the following prerequisites:
 
 To download and install the Mendix Service Console, follow these steps:
 
-1. Download the latest version of the Mendix Service Console by following the **Related downloads** link from the [Studio Pro Download Page](https://marketplace.mendix.com/link/studiopro/) of the Marketplace.
+1. Download the latest version of the [Windows Service Console](https://marketplace.mendix.com/link/component/223425) module from the Marketplace.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/on-premises-design/ms-windows/_index.md]: I note that in the Marketplace this is called **Windows Service Console**.
Should we change all the references to **Mendix Service Console** to be the current name, i.e. **Windows Service Console**?

**Review by MarkvanMents — COMMENTED** [review]
@yatindrakulkarni-mendix 
Thanks for the PR.
I've asked a question, below, about the name of the console. I think we should be consistent in what we call it.

File: content/en/docs/developerportal/deploy/on-premises-design/ms-windows/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/on-premises-design/ms-windows/_index.md]: Reverted in dd2e7b9180ff5d732a49a305d355aa0a65c9ab44

**Review by MarkvanMents — APPROVED** [review]
@yatindrakulkarni-mendix 
Thanks for the PR. Sorry about the delay - I forgot to check that the Marketplace component has been renamed.
Is it OK to publish this now?
Also, I notice that we use "Windows Service Console" in various places. Like here: https://docs.mendix.com/developerportal/deploy/security-checklist-for-your-on-premises-installation/#3-using-an-unprivileged-dedicated-user-account-for-every-application. Should I create a PR to update all these to also say "Mendix Service Console"?

Thanks for your patience.
Mark

**Comment by yatindrakulkarni-mendix** [comment]
@MarkvanMents 
We are yet to get latest logo for our marketplace module. Also, I need to confirm with the team regarding name change in all docs. Please hold to publish. Will let you know about the progress in upcoming weeks.

**Comment by MarkvanMents** [comment]
> @MarkvanMents We are yet to get latest logo for our marketplace module. Also, I need to confirm with the team regarding name change in all docs. Please hold to publish. Will let you know about the progress in upcoming weeks.

Thanks for the update.
I'll put a future label on this so it doesn't get published accidentally.

**Comment by MarkvanMents** [comment]
Approval given over Slack. Will merge once I have a clear build.

**Pull request merged**